src/components/review-block.tsx

```diff
@@ -0,0 +1,40 @@
+import ReviewsList from './reviews-list';
+import { ReviewsType } from '../types/reviews';
+
+import { useState } from 'react';
+
+type ReviewBlockProps = {
+  reviews: ReviewsType;
+}
+
+function ReviewBlock({reviews}: ReviewBlockProps): JSX.Element {
+  const [quantity, setQuantity] = useState(3);
+
+  function quantityCommentsDisplayed () {
```

Comment:
```suggestion
  function DisplayedComments Count() {
```
В названии не указано что это количество